src/index.ts

```diff
@@ -1,7 +1,8 @@
 import { Probot } from "probot";
 import fs from 'fs';
-import k8s, { V1Secret } from '@kubernetes/client-node';
+import  { V1Secret } from '@kubernetes/client-node';
 const { createAppAuth } = require("@octokit/auth-app");
+const k8s = require('@kubernetes/client-node');
```

Comment:
I had to do this since I couldn't build/start the app but I am not sure if its a problem just on my side.